bin/release-to-branch.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+set -e
+
+CURRENT_BRANCH_NAME=$(git rev-parse --abbrev-ref HEAD)
+BRANCH_NAME="pre-release-$CURRENT_BRANCH_NAME"
+
+# Remove local branch is it already exists
+if [ `git branch --list $BRANCH_NAME` ]; then
```

Comment:
Given the effect is the same, could we just force push the branch instead?

Reply:
I had issues at the subtree split stage when the branch already existed, but I can double check this.